pom.xml

```diff
@@ -293,6 +298,12 @@
             <version>3.1.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Siden vi gjorde om noen filer til kotlin, trengte vi mockito-kotlin for noen tester